.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -27,6 +27,7 @@ _Put an `x` in the boxes that apply. You can also fill these out after creating
 - [ ] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
 - [ ] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
 - [ ] PR Description is included
+- [ ] gif or screenshot is included if visual changes are made
```

Comment:
I think this is a good idea! It would definitely make some PRs easier to review and help expedite the review process for those PRs.

Just for clarity and to make the capitalization of the list items more consistent, how about rewording to:
> Screenshot or gif is included in PR description if visual changes are made

@jellypuno I agree with posting this in the contributor guidelines, as well. Who would be responsible for doing that?

Reply:
I was going to say that it could've been done in this same PR by updating the [Contributin.md](https://github.com/zowe/vscode-extension-for-zowe/blob/master/CONTRIBUTING.md) file, but we've never updated that file (personally I think we should since that's the default recommendation by GitHub when people open PRs)
We can tackle that as a documentation GH issue, and update the [wiki information](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) with that.